src/lib.rs

```diff
@@ -540,6 +540,14 @@ mod tests {
 		assert_eq!(amb.iter().collect::<Vec<_>>(), present_vec);
 	}
 
+	#[cfg(feature = "rand")]
+	#[test]
+	fn test_generate() {
+		let _ = Mnemonic::generate(24).unwrap();
```

Comment:
Maybe these could be doctests, so that they are also improving docs (not everyone might be aware of `rand_core`'s relation to `rand`)?

Reply:
Otoh, only one of these is really interesting …

Reply:
Hmm, good point :D will do